packages/swing-store-lmdb/src/snapStore.js

```diff
@@ -119,6 +125,8 @@ export function makeSnapStore(
    * @template T
    */
   async function load(hash, loadRaw) {
+    assert(!hash.includes('/'));
```

Comment:
maybe also `assert.typeof(hash, 'string')`, since if I was attacking this I'd fake both `.includes` and `.toString`